packages/overlay/test/overlay-trigger-click.test.ts

```diff
@@ -174,4 +177,37 @@ describe('Overlay Trigger - Click', () => {
         );
         expect(el.open).to.equal('click');
     });
+
+    it('opens with a delay on click', async () => {
+        const openedSpy = spy();
+        const closedSpy = spy();
+
+        const start = performance.now();
+
+        const el = await fixture<OverlayTrigger>(html`
+            <overlay-trigger
+                placement="right-start"
+                @sp-opened=${() => openedSpy()}
+                @sp-closed=${() => closedSpy()}
+            >
+                <sp-button slot="trigger" variant="primary"></sp-button>
+                <sp-tooltip
+                    slot="click-content"
+                    id="content"
+                    delayed
+                ></sp-tooltip>
+            </overlay-trigger>
+        `);
+
+        await elementUpdated(el);
+        el.setAttribute('open', 'click');
+        await waitUntil(
+            () => openedSpy.calledOnce,
+            'hover content projected to overlay',
+            { timeout: 8000 }
+        );
```

Comment:
```suggestion
        const start = performance.now();
        const opened = oneEvent(el, 'sp-opened');
        el.setAttribute('open', 'click');
        await opened;
```

If you structure the test like this you can get away without either of the spies and can be sure to only test the open workflow, rather than any delay that might come from the generation of the fixture.